hilti/toolchain/include/ast/declarations/parameter.h

```diff
@@ -133,4 +133,10 @@ inline bool areEquivalent(Parameter* p1, Parameter* p2) {
     return type::same(p1->type(), p2->type());
 }
 
+/** Returns true if two sets of parameters are equivalent, regardless of their ID. */
+inline bool areEquivalent(node::Set<Parameter> params1, node::Set<Parameter> params2) {
```

Comment:
No need for copies here.

```suggestion
inline bool areEquivalent(const node::Set<Parameter>& params1, const node::Set<Parameter>& params2) {
```